.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing notes
+
+## Pre-requirements: 
+git, make, curl, go, gcc, nginx, mkcert
+
+## Local setup
+Run `make -C api init` to install dependencies.
+
+#### To run nginx 
+Install latest nginx https://www.linuxbabe.com/ubuntu/install-nginx-latest-version-ubuntu-18-04  
```

Comment:
That's too Ubuntu-specific. "Install latest nginx" is good enough.

Reply:
Fixed